src/components/Dashboard/Dashboard.scss

```diff
@@ -1,17 +1,23 @@
 @import '../../index.scss';
 
-.viewport-height {
-    height: 90vh;
+.dashboard {
     width: 100%;
+    height: 100%;
 
-    &:focus {
-        outline: none;
+    .viewport-height {
+        height: calc(100% - 84px);

```

Comment:
this magic number is always used?

Reply:
Yes, it is because dashboard header has defined height of 84px

Reply:
Ok :)